science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -0,0 +1,237 @@
+
+
+## 1 Das Wichtigste in Kürze: Zahlen, Daten, Fakten
+
+- Ob Antigen-Schnelltest oder PCR-Test: der Positivenanteil von roten Warnungen ist stets größer als derjenige von grünen Warnungen (niedriges Risiko mit Risikobegegnung). Mit anderen Worten: dort wo Risikobegegnungen erkannt werden, kann die CWA das Risiko zuverlässig einschätzen.
+
+- Für Antigen-Schnelltests ist der Positivenanteil (der Anteil positiver an allen durchgeführten Tests) nach einer roten Warnung zu allen Zeiten größer als die Positivenanteile nach einer grünen Warnung (niedriges Risiko mit Risikobegegnung) bzw. bei niedrigem Risiko ohne Risikobegegnung.
+
+- Für PCR-Tests ist der Positivenanteil nach einer roten Warnung stets größer als der Positivenanteil nach einer grünen Warnung (niedriges Risiko mit Risikobegegnung). In Zeiten einer hohen Inzidenz ist der Positivenanteil nach einer roten Warnung zum Teil kleiner als bei niedrigem Risiko ohne Risikobegegnung – hier ist eine starke Vorselektion durch positive Antigen-Schnelltests zu vermuten.
+
+- Der erhöhte Positivenanteil nach einer roten Warnung im Vergleich zu den Positivenanteilen bei einem niedrigen Risiko mit bzw. ohne Risikobegegnung lässt sich nicht allein mit lokalen Unterschieden der Inzidenz erklären.
+
+- Für Antigen-Schnelltests und PCR-Tests ist die Abhängigkeit eines positiven Testergebnisses von der Inzidenz nach einer roten Warnung nur etwa halb so hoch, wie bei einem niedrigen Risiko ohne Risikobegegnung.
+
+- Die Unterschiede zwischen den Positivenanteilen und damit die Aussagekraft einer roten Warnung hinsichtlich des Infektionsrisikos sind in ländlichen Gebieten deutlicher als in urbanen.
+
+## 2 Positivenanteile (bundesweit)
+
+Täglich spenden etwa 15,4 Mio. Nutzende der CWA ihre Daten. So wurden bisher ca. 37,4 Mio. Testergebnisse übermittelt (Stand: 14. November 2022). Hierbei wurden nur solche Testergebnisse berücksichtigt, die positiv oder negativ ausfielen (keine ausstehenden und ungültigen). Durch die CWA identifizierte Risikobegegnungen wurden nur berücksichtigt, wenn sie bis zu 10 Tage vor der Testregistrierung lag (dies war bei ca. 10,7 Mio. Testergebnissen der Fall). Eine Angabe zum Kreis (Landkreis bzw. kreisfreie Stadt des Wohnorts des CWA-Nutzenden) lag bei ca. 14,3 Mio. Testergebnissen vor.
+
+### 2.1 RAT
+
+Insgesamt gab es ca. 27,4 Mio. übermittelte Ergebnisse von Antigen-Schnelltests (RAT), bei denen ca. 7,9 Mio. mindestens eine Risikobegegnung vorausging. In der folgenden Abbildung ist der zeitliche Verlauf der Positivenanteile für jede Art der Risikobenachrichtigung angegeben. Über die Datenspende werden erst seit dem 24. Juni 2021 Antigen-Schnelltest-Ergebnisse registriert.
+
+<img src="./ppa_risk_infection_ct_timeline_rat-1.png" class="figure-img img-fluid" alt="Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT)."/>
+<figcaption aria-hidden="true">Abb. 1: Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT).</figcaption>
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption `<figcaption>` requires `<figure>` as parent.

Reply:
Suggestion:

1. Define `.figure-img` in [src/assets/scss/_science.scss](https://github.com/corona-warn-app/cwa-website/blob/master/src/assets/scss/_science.scss)

```css
.figure-img {
    overflow: auto;
}
```

2. Wrap `<img>` with
```html
<figure class="figure-img">
...
</figure>
```

3. Remove `figure-img` from `<img>` tag

4. Remove `<br/>` tag

Result for example:

```html
<figure class="figure-img">
<img src="./ppa_risk_infection_ct_timeline_rat-1.png" class="img-fluid" alt="Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT)."/>
<figcaption aria-hidden="true">Abb. 1: Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT).</figcaption>
</figure>
```